packages/motion-config/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@hashicorp/react-motion-config",
+  "version": "0.1.0",
+  "description": "",
+  "author": "HashiCorp",
+  "license": "MPL-2.0",
+  "dependencies": {
+    "framer-motion": "^6.3.11"
```

Comment:
I wonder if we want to make this a peer dep? 🤔 Since there is a chance we could end up with two versions depending on what version the consumer has installed

Reply:
That is a good callout, I would assume so based on my limited understanding and what I am reading.